dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/export/openaire/FirstNames.java

```diff
@@ -0,0 +1,263 @@
+package edu.harvard.iq.dataverse.export.openaire;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.logging.Logger;
+import java.util.HashMap;
+import java.util.logging.Level;
+
+/**
+ *
+ * @author francesco.cadili@4science.it
+ */
+public class FirstNames {
+
+    private static FirstNames instance = null;
+
+    private final int capacity = (int) ((49000 + 33000) * 1.2 * 0.8);
+    private final boolean loadNameDict = true;
+    private final boolean loadYobDict = true;
+    private final boolean loadHint = true;
+    private final int nameDictStart = 2;
+    private final int nameDictLength = 27;
+    private int duplicates = 0;     // 20%, factor 0.8
+
+    private HashMap<String, String> map = new HashMap<String, String>(capacity);
+    private static final Logger logger = Logger.getLogger(FirstNames.class.getCanonicalName());
+    public static String NAME_DICT_FILENAME = "edu/harvard/iq/dataverse/firstNames/nam_dict.txt";
+    public static String YOB_FILENAME = "edu/harvard/iq/dataverse/firstNames/yob2017.txt";
```

Comment:
There is no point in keeping `nam_dict.txt` and `yob2017.txt` in `src/test/resources` and `src/main/resources`